tests/Tests/200__monitor_and_manage/200__metrics/201__billing_metrics.robot

```diff
@@ -34,105 +84,191 @@ Begin Billing Metrics Web Test
   RHOSi Setup
 
 End Web Billing Metrics Test
-  CleanUp JupyterHub
-  SeleniumLibrary.Close All Browsers
+    CleanUp JupyterHub
+    SeleniumLibrary.Close All Browsers
+
+Test Setup For Matrics Web Test
+    Set Library Search Order    SeleniumLibrary
+    Open Browser    ${OCP_CONSOLE_URL}    browser=${BROWSER.NAME}    options=${BROWSER.OPTIONS}
+    Login To OCP
+    Wait Until OpenShift Console Is Loaded
+    Click Button    Observe
+    Click Link    Metrics
+    Wait Until Element is Visible    xpath://textarea[@class="pf-c-form-control query-browser__query-input"]
+
+Test Teardown For Matrics Web Test
+    SeleniumLibrary.Close All Browsers
+
+Run Query On Metrics And Return Value
+    [Documentation]    Fires query in metrics through web browser and returns value
+    [Arguments]    ${query}    ${count_of_columns}    # count of columns + 1 like name,values example: ${count_of_columns}=3
+    Input Text    xpath://textarea[@class="pf-c-form-control query-browser__query-input"]    ${query}
+    Click Button    Run queries
+    Wait Until Element is Visible    xpath://table[@class="pf-c-table pf-m-compact"]    timeout=15seconds
+    @{data} =    Get WebElements    //table[@class="pf-c-table pf-m-compact"] //tbody/tr/td[${count_of_columns}]
+    [Return]    ${data[0].text}
+
+Fire Query On Prometheus And Return Value
```

Comment:
is it the RHODS prometheus right? maybe it is worth to add "RHODS" in the kw name, to differentiate it from the Openshift prometheus

Reply:
yes
Thanks